prac_05/word_occurrences.py

```diff
@@ -1 +1,20 @@
-fdhryrf
\ No newline at end of file
+"""
+Word Occurrences
+Estimate: 1 hour 10 minutes
+Actual: 48 minutes
+"""
+
+word_to_count = {}
+text = input("Text: ")
+words = text.split()
+for word in words:
+    word_to_count[word] = word_to_count.get(word, 0) + 1
+
+words = list(word_to_count.keys())
+words.sort()
+
+max_length = max(len(word) for word in words)
+for word in words:
+    print(f"{word:{max_length}} : {word_to_count[word]}")
+
+
```

Comment:
You could add some comments on some lines to make readers understand what it does